scripts/config/syncAllTemplates.mjs

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+/** 
+* This script is for the purpose of keeping a component repository in sync with the latest set of workflows defined for Auro components. 
+* Running this script will delete all workflows that exist in the repository and replace them with the latest defined workflows for Auro components. **/
+
+import AuroWorkflows from './syncWorkflows.mjs';
```

Comment:
None of these files will be in the npm package due to this 

https://github.com/AlaskaAirlines/auro-library/blob/9a6a7d05783fd1f3be5cd06e75b2bf57effab83f/.npmignore#L2